config/install/field.field.media.video.field_video.yml

```diff
@@ -11,11 +11,14 @@ field_name: field_video
 entity_type: media
 bundle: video
 label: 'Video URL'
-description: ''
+description: 'Enter a YouTube/Vimeo video URL. <a href="https://www.drupal.org/project/video_embed_field">Click here</a> to add support for more providers.'
```

Comment:
This is not true... You won't be able to add more providers on that page. You will just get info about how to do that.